orttraining/pytorch_frontend_examples/nvidia-bert/Onnxrt.dockerfile

```diff
@@ -0,0 +1,74 @@
+# FROM nvcr.io/nvidia/tritonserver:20.03-py3-clientsdk as trt
+FROM mcr.microsoft.com/azureml/base-gpu:openmpi3.1.2-cuda10.1-cudnn7-ubuntu18.04
+
+ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/opt/cmake/bin:/opt/miniconda/bin:${PATH}
+ENV LD_LIBRARY_PATH /opt/miniconda/lib:$LD_LIBRARY_PATH
+
+RUN apt update -y && pip install --upgrade pip
+
+# install pytorch stable using CUDA 10.1
+RUN pip install torch==1.5.0+cu101 torchvision==0.6.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
+
+# add pytorch patch for opset 10
+COPY pyt_patch /tmp/pyt_patch
```

Comment:
Is this a reproducible step if someone wants to build a customer Docker image derived from this?